.changeset/cuddly-dodos-exercise.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/slang": patch
+---
+
+Fix parsing `override` identifiers before 0.6.0
```

Comment:
My understanding is that `enabled` should not affect parsing identfiers, only `reserved` should. Not sure I understand how this is affected by this PR?